operator/pkg/clustermesh/clustermesh.go

```diff
@@ -44,16 +50,16 @@ type clusterMesh struct {
 	endpointSliceInformerFactory informers.SharedInformerFactory
 }
 
-func newClusterMesh(lc cell.Lifecycle, params clusterMeshParams) *clusterMesh {
+func newClusterMesh(lc cell.Lifecycle, params clusterMeshParams) (*clusterMesh, error) {
```

Comment:
I guess that the additional error parameter could be reverted, as always nil.

Reply:
Woops indeed :+1: